### DISCUSSION TITLE
any plan for freetype font support?

### DISCUSSION THREAD

**Discussion by dotnfc** [lovyan03/LovyanGFX#193] [Q&A]
i see repo has lgfx_TTFfont.*, but has not impl-ed.
any plan for this?

**Comment by lovyan03** [comment]
I would like to implement TTF support, but the amount of work involved has proven to be tremendous, so work has been halted for now.

**Comment by dotnfc** [comment]
someone on youtube channel shared a video, this maybe a startpoint:

https://www.youtube.com/watch?v=Wql4WTpPrWs
https://github.com/mireq/esp32-st7789-demo/

**Comment by lovyan03** [comment]
We don't intend to do that, because if we include freetype, we will have to change the license.

**Comment by dotnfc** [comment]
i've tested 'https://github.com/takkaO/OpenFontRender', it works very well with lovyanGfx, although it used freetype2.

**Reply by lovyan03** [reply]
yes, I know it.

**Comment by dotnfc** [comment]
lovyanGFX has irc channel?

**Reply by lovyan03** [reply]
no
